tests/unit/script/htmlimageelement.rs

```diff
@@ -0,0 +1,16 @@
+
+use script::test::sizes::parse_a_sizes_attribute;
+use script::test::DOMString;
+
+#[test]
+fn empty_vector() {
+    assert!(parse_a_sizes_attribute(DOMString::new(), None).is_ok());
+} 
+
+#[test]
+fn test_whitespace() {
+    assert!(parse_a_sizes_attribute(DOMString::from("     (min-width: 500px)"),
+            None).is_ok());
```

Comment:
Let's think about what this test is checking. `(min-width: 500px)` by itself is not a valid size; it is a media query, and the sizes algorithm then tries to parse a length that follows it. Since there is no length present, this string would be considered a parse error. That's fine, but we should also verify in a separate test that a valid size with extra whitespace parses correctly and gives us a real parsed size.